index.html

```diff
@@ -1351,18 +1351,15 @@ <h2 class="h2-title">Subscribe our newsletter</h2>
                                     <div class="footer-info">
                                         <div class="footer-logo">
                                             <a href="index.html">
-                                                <img src="logo1.png" alt="">
+                                                <img src="logos.png" alt="">
```

Comment:
cross check whether the logo used is the one without the black stroke, or the one that is used in the nav bar has to be used in the footer too.

Reply:
yeah its the same logo but i have just adjusted the size of the logo used in the footer for better visibility